project_task_description_template/models/task_template.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2023 Cetmix OÜ
```

Comment:
**nitpick:** the name of this file should be the same as the name of the model: `project_task_template.py`

Reply:
Thanks, I'll fix it